shell/platform/linux/fl_renderer.h

```diff
@@ -73,6 +101,15 @@ GdkVisual* fl_renderer_get_visual(FlRenderer* self,
                                   GdkScreen* screen,
                                   GError** error);
 
+/**
+ * fl_renderer_set_window:
+ * @renderer: an #FlRenderer.
+ * @window: the GDK Window this renderer will render to.
+ *
+ * Set the window this renderer will use.
+ */
+void fl_renderer_set_window(FlRenderer* self, GdkWindow* window);
```

Comment:
self -> renderer.

Reply:
Some exisiting code is wrong here, see https://github.com/flutter/engine/pull/20163